### ISSUE TITLE
CICD Pipelines - stagedAutoprefixer is not a function

### ISSUE THREAD

**Issue by Smeethy** [csstools/postcss-preset-env#217]
Hello,

Tl;dr, I face some issue transpiling my CRA ReactJS application using an AZ Devops pipeline:
**./src/index.css
TypeError: stagedAutoprefixer is not a function**

A bit of context:
- It is based on the template Create-react-app v4
- We're using Tailwind so we're generating the css to import using postcss 8
- We used the yield command on the cra template to access the configuration files
- It transpiles properly on a dev machine to a production build

The packages.json

```json
{
  "name": "sources",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "@babel/core": "7.12.3",
    "@babel/plugin-proposal-class-properties": "^7.14.5",
    "@babel/preset-typescript": "^7.15.0",
    "@fluentui/react": "^8.34.4",
    "@fluentui/react-northstar": "^0.58.0",
    "@microsoft/applicationinsights-react-js": "^3.2.0",
    "@microsoft/applicationinsights-web": "^2.7.0",
    "@microsoft/teams-js": "^1.10.0",
    "@pmmmwh/react-refresh-webpack-plugin": "0.4.2",
    "@svgr/webpack": "5.4.0",
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "@types/jest": "^26.0.15",
    "@types/node": "^12.20.25",
    "@types/react": "^17.0.21",
    "@types/react-dom": "^17.0.0",
    "@typescript-eslint/eslint-plugin": "^4.5.0",
    "@typescript-eslint/parser": "^4.5.0",
    "@uifabric/icons": "^7.6.0",
    "archiver": "^5.3.0",
    "axios": "^0.21.4",
    "babel-eslint": "^10.1.0",
    "babel-jest": "^26.6.0",
    "babel-loader": "8.1.0",
    "babel-plugin-named-asset-import": "^0.3.7",
    "babel-plugin-transform-typescript-metadata": "^0.3.2",
    "babel-preset-react-app": "^10.0.0",
    "bfj": "^7.0.2",
    "camelcase": "^6.1.0",
    "case-sensitive-paths-webpack-plugin": "2.3.0",
    "colorthief": "^2.3.2",
    "cross-env": "^7.0.3",
    "css-loader": "4.3.0",
    "dotenv": "8.2.0",
    "dotenv-expand": "5.1.0",
    "env-cmd": "^10.1.0",
    "eslint": "^7.11.0",
    "eslint-config-react-app": "^6.0.0",
    "eslint-plugin-flowtype": "^5.2.0",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-jest": "^24.1.0",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-react": "^7.21.5",
    "eslint-plugin-react-hooks": "^4.2.0",
    "eslint-plugin-testing-library": "^3.9.2",
    "eslint-webpack-plugin": "^2.1.0",
    "file-loader": "6.1.1",
    "fs-extra": "^9.0.1",
    "html-webpack-plugin": "4.5.0",
    "i18next": "^21.1.0",
    "i18next-browser-languagedetector": "^6.1.2",
    "identity-obj-proxy": "3.0.0",
    "inversify": "^5.1.1",
    "inversify-inject-decorators": "^3.1.0",
    "invert-color": "^2.0.0",
    "jest": "26.6.0",
    "jest-circus": "26.6.0",
    "jest-resolve": "26.6.0",
    "jest-watch-typeahead": "0.6.1",
    "jwt-decode": "^3.1.2",
    "mini-css-extract-plugin": "0.11.3",
    "moment": "^2.29.1",
    "msteams-react-base-component": "^3.1.0",
    "office-ui-fabric-react": "^7.176.1",
    "optimize-css-assets-webpack-plugin": "5.0.4",
    "pnp-webpack-plugin": "1.6.4",
    "postcss-cli": "^9.0.1",
    "postcss-flexbugs-fixes": "4.2.1",
    "postcss-loader": "3.0.0",
    "postcss-normalize": "8.0.1",
    "postcss-preset-env": "6.7.0",
    "postcss-safe-parser": "5.0.2",
    "prompts": "2.4.0",
    "qs": "^6.10.1",
    "react": "^17.0.2",
    "react-accessible-accordion": "^3.3.5",
    "react-app-polyfill": "^2.0.0",
    "react-beautiful-dnd": "^13.1.0",
    "react-collapsible": "^2.8.4",
    "react-dev-utils": "^11.0.1",
    "react-dom": "^17.0.2",
    "react-i18next": "^11.12.0",
    "react-infinite-scroll-component": "^6.1.0",
    "react-redux": "^7.2.5",
    "react-refresh": "^0.8.3",
    "react-router-dom": "^5.3.0",
    "react-slick": "^0.28.1",
    "react-tracking": "^9.0.0",
    "redux-devtools-extension": "^2.13.9",
    "redux-thunk": "^2.3.0",
    "reflect-metadata": "^0.1.13",
    "resolve": "1.18.1",
    "resolve-url-loader": "^3.1.2",
    "sass-loader": "8.0.2",
    "semver": "7.3.2",
    "slick-carousel": "^1.8.1",
    "style-loader": "1.3.0",
    "terser-webpack-plugin": "4.2.3",
    "ts-pnp": "1.2.0",
    "typescript": "^4.1.2",
    "url-loader": "4.1.1",
    "web-vitals": "^1.0.1",
    "webpack": "4.44.2",
    "webpack-dev-server": "3.11.0",
    "webpack-manifest-plugin": "2.2.0",
    "workbox-webpack-plugin": "5.1.4"
  },
  "scripts": {
    "start": "node scripts/start.js",
    "build-dev:tailwind": "cross-env NODE_ENV=production postcss src/tailwind.css -o src/tailwind.generated.css",
    "prebuild-dev": "npm run build-dev:tailwind",
    "build-dev": "env-cmd -f .env cross-env NODE_ENV=development node scripts/build.js",
    "test": "node scripts/test.js"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@babel/plugin-proposal-decorators": "^7.15.4",
    "@tailwindcss/line-clamp": "^0.2.1",
    "@types/react-router-dom": "^5.3.0",
    "@types/react-tracking": "^8.1.1",
    "autoprefixer": "^10.3.6",
    "babel-plugin-parameter-decorator": "^1.0.16",
    "postcss": "^8.3.8",
    "tailwindcss": "^2.2.16"
  },
  "jest": {
    "roots": [
      "<rootDir>/src"
    ],
    "collectCoverageFrom": [
      "src/**/*.{js,jsx,ts,tsx}",
      "!src/**/*.d.ts"
    ],
    "setupFiles": [
      "react-app-polyfill/jsdom"
    ],
    "setupFilesAfterEnv": [
      "<rootDir>/src/setupTests.ts"
    ],
    "testMatch": [
      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
      "<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}"
    ],
    "testEnvironment": "jsdom",
    "testRunner": "C:\\Users\\laurent.chbibi\\Desktop\\Homepage\\LOreal-BAWorkplace-Home\\src\\OneRetail-HomeSPA\\sources\\node_modules\\jest-circus\\runner.js",
    "transform": {
      "^.+\\.(js|jsx|mjs|cjs|ts|tsx)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$",
      "^.+\\.module\\.(css|sass|scss)$"
    ],
    "modulePaths": [],
    "moduleNameMapper": {
      "^react-native$": "react-native-web",
      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
    },
    "moduleFileExtensions": [
      "web.js",
      "js",
      "web.ts",
      "ts",
      "web.tsx",
      "tsx",
      "json",
      "web.jsx",
      "jsx",
      "node"
    ],
    "watchPlugins": [
      "jest-watch-typeahead/filename",
      "jest-watch-typeahead/testname"
    ],
    "resetMocks": true
  },
  "babel": {
    "presets": [
      "react-app"
    ]
  }
}
```

The error:
![pipeline](https://user-images.githubusercontent.com/45082399/138438586-73ca71e2-7819-439b-acbc-6c7cb4d581c1.JPG)

I am mainly looking for ideas to why it would selectively fails on a pipeline while working locally, if you have any.

Best Regards,
S

**Comment by Antonio-Laguna** [comment]
@Smeethy could you try with the new release?

**Comment by Antonio-Laguna** [comment]
This repository is getting archived. If this issue keeps happening please re-open on [@csstools/postcss-plugins](https://github.com/csstools/postcss-plugins/issues/new/choose) and we'll do our best to fix it!
